### PR TITLE
Convertion of partials to full imports

### PR DESCRIPTION
**Pull request by rarila**
I finally have converted "all" partial to full imports.

I was looking at all cases but I don’t think readability suffers at any case, as mostly they are unambiguous due prefixing or postfixing or just by naming or context.

One exception is the ``PhpParser`` namespace which will need an extra issue for discussion and so is left out here.

### CODE REVIEW

**Comment by rarila** [comment]
I guess the error is related to #7153?!

File: src/Psalm/Codebase.php [hunk 1]
```diff
@@ -1124,7 +1136,7 @@ public function getSymbolLocation(string $file_path, string $symbol): ?CodeLocat
 
             $file_contents = $this->getFileContents($file_path);
 
-            return new CodeLocation\Raw(
+            return new Raw(
```
Comment (weirdan) [inline src/Psalm/Codebase.php:1139]: This is an example of where the readability does suffer. Could be renamed to `RawLocation` though.

**Comment by weirdan** [comment]
I'm halfway through the review, and this is actually not too bad.

**Review by weirdan — APPROVED** [review]
Note to self: review ParseTree, UnresolvedConstant and CodeLocation class naming.

File: src/Psalm/Codebase.php (hunk 1 above)
Comment (rarila) [reply src/Psalm/Codebase.php:1139]: Yep, ``Raw`` is quite inexpressively.
The good thing is, it is used only 4 times. Anyway, ``DocblockTypeLocation`` and ``ParseErrorLocation`` are postfixed, so this should really be ``RawLocation``.

But I think this should go into an extra PR.

**Comment by rarila** [comment]
> Note to self: review ParseTree, UnresolvedConstant and CodeLocation class naming.

Those two should probably also be added to that list:
```
Psalm\Internal\LanguageServer\Client\TextDocument // => ClientTextDocument ?!
Psalm\Internal\LanguageServer\Server\TextDocument // => ServerTextDocument ?!
```

File: src/Psalm/Codebase.php (hunk 1 above)
Comment (weirdan) [reply src/Psalm/Codebase.php:1139]: > But I think this should go into an extra PR.

Of course.

**Comment by orklah** [comment]
Thanks!

**Pull request merged**